app/code/Magento/Sales/Block/Adminhtml/Order/Create/Form.php

```diff
@@ -120,7 +120,7 @@ public function getSaveUrl()
     public function getCustomerSelectorDisplay()
     {
         $customerId = $this->getCustomerId();
-        if ($customerId === null) {
+        if (empty($customerId)) {
```

Comment:
I guess the usage of `empty()` function in redundant in this case, considering the variable always exists.
The issue caused by zero `0` value that does not match the condition.
So, I would recommend simplifying this condition to:
```
if (!$customerId) {
```

Reply:
I have refactored the if condition